nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml

```diff
@@ -0,0 +1,150 @@
+- scm:
+    name: nfs-ganesha
+    scm:
+      - git:
+          url: https://github.com/nfs-ganesha/nfs-ganesha.git
+          branches:
+            - $NFS_GANESHA_BRANCH
+          skip-tag: true
+          wipe-workspace: true
+          basedir: "nfs-ganesha"
+
+- scm:
+    name: nfs-ganesha-debian
+    scm:
+      - git:
+          url: https://github.com/nfs-ganesha/nfs-ganesha-debian.git
+          branches:
+            - xenial-nfs-ganesha-ceph-fsals
+          skip-tag: true
+          wipe-workspace: true
+          basedir: "nfs-ganesha-debian"
+- job:
+    name: nfs-ganesha
+    project-type: matrix
+    defaults: global
+    display-name: 'nfs-ganesha'
+    block-downstream: false
+    block-upstream: false
+    properties:
+      - github:
+          url: https://github.com/nfs-ganesha/nfs-ganesha
+    concurrent: true
+    parameters:
+      - string:
+          name: NFS_GANESHA_BRANCH
+          description: "The git branch (or tag) to build"
+          default: "V2.5-stable"
```

Comment:
it isn't really hard coding, just defaulting to one if it is not defined, which I think is good for this use case. We do have SHA1s for luminous that are released but those are "rc" releases.

These are the current SHA1s that we've released: https://chacra.ceph.com/r/ceph/luminous/